distribution/docs/src/main/resources/_contents/_developing/developing-intro-contents.adoc

```diff
@@ -0,0 +1,816 @@
+
+== ${branding} Development Guidelines
+
+[NOTE]
+====
+Development requires full knowledge of the ${branding} Catalog.
+====
+
+${branding} is written in Java and requires a moderate amount of experience with the Java programming language, along with Java terminology, such as packages, methods, classes, and interfaces.
+${branding} uses a small OSGi runtime to deploy components and applications.
+Before developing for ${branding}, it is necessary that developers have general knowledge on OSGi and the concepts used within.
+This includes, but is not limited to, Catalog Commands and the following topics:
+
+* The Service Registry
+** How services are registered
+** How to retrieve service references
+* Bundles
+** Their role in OSGi
+** How they are developed
+
+Documentation on OSGi can be viewed at the http://www.osgi.org[OSGi Alliance website].
+Helpful literature for beginners includes _OSGi and Apache Felix 3.0 Beginner's Guide_ by Walid Joseph Gédéon and _OSGi in Action: Creating Modular Applications in Java_ by Richard Hall, Karl Pauls, Stuart McCulloch, and David Savage.
+
+=== Recommended Hardware
+
+Because of its modular nature, ${branding} may require a few or many system resources, depending on which bundles and features are deployed.
+In general, ${branding} will take advantage of available memory and processors.
+A 64-bit JVM is required, and a typical installation is performed on a single machine with 16GB of memory and eight processor cores.
```

Comment:
OpenJDK should have all of the security things you'd want already installed. Oracle does require you to use the JCE Unlimited Strength policies in their JDK if you go that route, but it isn't required out of the box. If you try to use 256bit ciphers on Oracle JDK, you'll need the policy.